src/app/project/ontology/ontology.component.html

```diff
@@ -1,5 +1,7 @@
 <div *ngIf="projectAdmin" class="desktop-only">
 
+    <p class="note warning center">This is a first version of the ontology editor. Not all features are implemented yet.
+        Be careful when using it!</p>
```

Comment:
"Be careful when using it!": this turn of phrase is quite scary for the user...

Reply:
As you know, we can always change my english sentences. I'm open for suggestions 😉

Reply:
"use it on your own risk" 😂
No, just kidding. I thought it would be good to give a hint, that we know about the improvements and it's still work in progress. But it would be good to have an ontology create to start with simple ontologies on prod server. I need this tool for some projects in January...

Reply:
"Some features may not work as intended" is usually the corporate way of saying "Please don't be an idiot" 😂

Reply:
Done in 122c8b8